corehq/apps/enterprise/tests/test_tasks.py

```diff
@@ -110,31 +110,32 @@ def test_auto_deactivate_mobile_workers_task(
         metrics_gauge
     ):
         auto_deactivate_mobile_workers()
-        domains_checked_for_inactivity = [
+        domains_checked_for_inactivity = set(
             arg[0][0]
             for arg in deactivate_mobile_workers_by_inactivity.call_args_list
-        ]
-        self.assertListEqual(
-            domains_checked_for_inactivity,
-            [
-                'domain-account1-002',
-                'domain-account1-001',
-                'domain-account2-003',
-                'domain-account2-002',
-                'domain-account2-001',
-            ]
         )
-        domains_with_custom_deactivations = [
+        self.assertEqual(
```

Comment:
Can't you just use `assertEqual` on the two sets, which will use `assertSetEqual`, rather than computing the difference?